app/src/main/res/layout/include_omnibar_toolbar_mockup.xml

```diff
@@ -16,111 +16,101 @@
+            android:text="@string/omnibarInputHint"
+            android:textColor="?attr/daxColorSecondaryText" />
 
-            <com.duckduckgo.common.ui.view.text.DaxTextView
-                android:id="@+id/omnibarTextInputMockup"
-                android:layout_width="match_parent"
-                android:layout_height="match_parent"
-                android:gravity="start|center"
-                android:maxLines="1"
-                android:paddingStart="37dp"
-                android:paddingEnd="8dp"
-                android:text="@string/omnibarInputHint"
-                android:textColor="?attr/daxColorSecondaryText"/>
+    </FrameLayout>
 
-        </FrameLayout>
+    <FrameLayout
+        android:id="@+id/fireIconMenuMockup"
+        android:layout_width="wrap_content"
+        android:layout_height="match_parent"
+        android:paddingHorizontal="@dimen/keyline_2"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toStartOf="@id/tabsMenuMockup"
+        app:layout_constraintStart_toEndOf="@id/omniBarContainerMockup"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:visibility="visible">
 
-        <FrameLayout
-            android:id="@+id/fireIconMenuMockup"
+        <ImageView
+            android:id="@+id/fireIconImageViewMockup"
             android:layout_width="wrap_content"
-            android:layout_height="match_parent"
-            android:layout_marginStart="2dp"
-            android:paddingStart="4dp"
-            android:paddingEnd="4dp"
-            app:layout_constraintBottom_toBottomOf="parent"
-            app:layout_constraintEnd_toStartOf="@id/tabsMenuMockup"
-            app:layout_constraintStart_toEndOf="@id/omniBarContainerMockup"
-            app:layout_constraintTop_toTopOf="parent"
-            tools:visibility="visible">
+            android:layout_height="wrap_content"
+            android:layout_gravity="center"
+            android:background="?attr/selectableItemBackgroundBorderless"
+            android:contentDescription="@string/browserPopupMenu"
+            android:padding="@dimen/toolbarIconPadding"
+            android:src="@drawable/ic_fire" />
+    </FrameLayout>
 
-            <ImageView
-                android:id="@+id/fireIconImageViewMockup"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:layout_gravity="center"
-                android:background="?attr/selectableItemBackgroundBorderless"
-                android:contentDescription="@string/browserPopupMenu"
-                android:padding="@dimen/toolbarIconPadding"
-                android:src="@drawable/ic_fire"/>
-        </FrameLayout>
+    <com.duckduckgo.app.browser.TabSwitcherButton
+        android:id="@+id/tabsMenuMockup"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginHorizontal="@dimen/keyline_2"
```

Comment:
are these changes intentional? I was under the assumption that there wouldn’t be padding changes in this PR

Reply:
This just fixes the alignment issue with the mock toolbar. The padding changes will come in a different PR.